src/Sylius/Bundle/ApiBundle/Command/Cart/ChangeItemQuantityInCart.php

```diff
@@ -25,19 +25,11 @@ class ChangeItemQuantityInCart implements OrderTokenValueAwareInterface, Subreso
     /** @var string|null */
     public $orderItemId;
 
-    /**
-     * @psalm-immutable
-     *
-     * @var int
-     */
-    public $quantity;
-
-    public function __construct(int $quantity)
+    public function __construct(public ?int $quantity)
```

Comment:
As above 👆🏼

Reply:
I am not sure how it works in the previous comment, but here IIRC we have those fields mutable because of  [enricher](https://github.com/Sylius/Sylius/blob/1.13/src/Sylius/Bundle/ApiBundle/DataTransformer/OrderTokenValueAwareInputCommandDataTransformer.php)

Reply:
As above :)